src/app/pages/organization/companies/company/company.component.ts

```diff
@@ -24,6 +24,7 @@ import { Utils } from '../../../../utils/Utils';
 })
 export class CompanyComponent implements OnInit {
   @Input() public currentCompanyID!: string;
+  @Input() public canCreateCompany: boolean;
```

Comment:
Pass also canUpdateCompany.
No need to get it from the company in the HTTP request now.